packages/api-openfin/notification.html

```diff
@@ -0,0 +1,11 @@
+<h5 id="message">Message</h5>
```

Comment:
>OpenFin notifications are html pages in their own right, and this is just injected inside the body of that page.

Are you sure that this is just injected into the body? From the docs  and examples it looks like it's the entire HTML page.